deploy/bin/import-docker-worker-secrets

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
```

Comment:
I would like to get another set of eyes on this because of the security implications and also the use of password store (cc @djmitche) 

What concerns me here is that we are dropping secrets into this directory.  Wander said he would look into moving them elsewhere, but we also want to remove them afterwards so we do not have secrets floating around on people's machines.

Is there a way we could get these secrets from password store, and use them with the AMI generation, without having them dropped unencrypted on disk?

Reply:
We can remove them at the end of the build script, I didn't do it because it was not done before and I thought there was a good reason for that. Btw, I changed the patch to put those into `/tmp`.

Reply:
So one way around this would be to use `/dev/shm` for the sensitive files -- and be careful to put them in an 0700 directory and remove them afterward.